docs/components/table/README.md

```diff
@@ -382,12 +383,28 @@ will emit the `filtered` event, passing a single argument which is the complete
 items passing the filter routine. Treat this argument as read-only.
 
 ### Sorting
+As mentioned above in the **fields** section, you can make columns sortable. Clciking on
+sortable a column header will sort the column in ascending direction, while clicking
+on it again will switch the direction or sorting.  Clicking on a non-sortable column
```

Comment:
Although some may want to clear the sort... but you never know.  The original table (from before I started working on it) cleared the sort on a non-sortable column.

Some people may want to table to display in the order sent from the server (as not having a null sort by would case b-table to sort the data which is received).